specification/communication/data-plane/SipRouting/readme.ruby.md

```diff
@@ -7,6 +7,10 @@ package-name: azure_mgmt_communicationservices
 package-version: 2021-05-01-preview
 azure-arm: true
 ```
+### Tag: package-2022-09-01-preview and ruby
+
+These settings apply only when `--tag=package-2022-09-01-preview --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
```

Comment:
Do you need settings here?

Reply:
Hello, I put those settings for new version so it's possible to generate each version. Does that make sense?